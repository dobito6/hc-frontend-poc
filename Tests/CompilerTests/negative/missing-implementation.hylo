//! no-std

trait P {
  fun f0() {}
  fun f1()
  fun f2()
  fun f3()
}

extension Void {
  fun f1() {}
}

given Void: P {
  fun f2() {}
}
