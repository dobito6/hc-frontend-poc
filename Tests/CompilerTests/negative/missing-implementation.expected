negative/missing-implementation.hylo:14.1-6: error: no implementation of 'f3'
given Void: P {
~~~~~
